recipe/meta.yaml

```
@@ -7,11 +7,12 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_rev: db/scikit-build-core
+  git_url: https://github.com/TileDB-Inc/TileDB-Py.git
 
 build:
+  script: TILEDB_PATH=${PREFIX} {{ PYTHON }} -m pip install --no-build-isolation --no-deps --ignore-installed -v . # [not win]
+  script: set "TILEDB_PATH=${PREFIX}" && {{ PYTHON }} -m pip install --no-build-isolation --no-deps --ignore-installed -v . # [win]
```

```suggestion
  script: set "TILEDB_PATH=${PREFIX}" && {{ PYTHON }} -m pip install --no-build-isolation --no-deps --ignore-installed -v .  # [win]
```

Not important for this test PR, but FYI this is what the linter is complaining about. It wants two spaces between the end of the line and the start of the jinja2 selector